modules/service/src/main/scala/lucuma/odb/Main.scala

```diff
@@ -178,15 +179,15 @@ object FMain extends MainParams {
           .onFinalize(cell.updateAndGet(_ + 1).flatMap(n => Logger[F].debug(s"released session (should be $n remaining)")))
 
   /** A resource that yields a running HTTP server. */
-  def serverResource[F[_]: Async](
+  def serverResource[F[_]: Async: Network](
     port: Port,
     app:  WebSocketBuilder2[F] => HttpApp[F]
   ): Resource[F, Server] =
-    BlazeServerBuilder
-      .apply[F]
-      .bindHttp(port.value, "0.0.0.0")
+    EmberServerBuilder.default[F]
+      .withPort(port)
+      .withHost(host"0.0.0.0")
```

Comment:
I think it might be the shutdown timeout that adds 30 seconds to every test case. I believe I "solved" this by decreasing the default.  I don't know what the ramifications of that would be though.  Maybe test `.withShutdownTimeout(Duration.Zero)`?

Reply:
Yeah I think this was the fix for the tests. Locally tests take 5 seconds longer without this line with Ember (but not with Blaze). And with this Ember and blaze takes the same amount of time